internal/security/gnxi/rpcexec.go

```diff
@@ -641,7 +661,16 @@ func GribiModify(ctx context.Context, dut *ondatra.DUTDevice, opts []grpc.DialOp
 	if err != nil {
 		return err
 	}
-	_, err = mStream.Recv()
+	for {
+		msg, err := mStream.Recv()
+		if err == io.EOF {
+			return nil
+		} else if err != nil {
```

Comment:
why else? why not just switch on err ?
chains of tihs if/else/elseif/etc are toilsome to think though, when all you are relaly doing is a switch statement.

(also see previous comment about adminishment in golang for use of else generally)

Reply:
Replace if else section with switch case in new commit